### DISCUSSION TITLE
AccessDenied in scene.splinecode

### DISCUSSION THREAD

**Discussion by kaissaroj** [splinetool/react-spline#135] [Q&A]
I try to export scene.splinecode url  but getting AccessDenied issues. Is there any settings for this or am i missing anything here

`<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>JS63WF0ZEB68QNC7</RequestId>
<HostId>
VAsZho+gA3vvgCEKsgAtMCy02zCClbLwXrMtqZc9aOOBjfWQ+c9XITQfYIs493z5R7uCDMFAqts=
</HostId>
</Error>`

**Comment by Hamziss** [comment]
i have the same problem did you managed to find a fix for that?

**Comment by jtoy** [comment]
I get the same issues.
